docs/_layouts/docs.html

```diff
@@ -3,193 +3,227 @@
+                </a>
+            </li>
+        </ul>
+
+        <h5>Application Deployment</h5>
+        <ul class="list-unstyled">
+            <li>
+                <a href="{{ site.baseurl }}/docs/application-basics.html">
+                    Application Basics
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/deployments.html">
+                    Application Deployments
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/command-line-flags.html">
+                    Command Line Flags
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/developing-vm.html">
+                    Run a Local Version of Marathon
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/native-docker-private-registry.html">
+                    Using a Private Docker Registry
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/artifact-store.html">
+                    Artifact Store
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/constraints.html">
+                    Constraints
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/event-bus.html">
+                    Event Bus
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/health-checks.html">
+                    Health Checks
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/ip-per-task.html">
+                    IP-per-task
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/task-environment-vars.html">
+                    Task Environment Variables
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/">
+                    Service Discovery and Load Balancing
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/ports.html">
+                    Ports
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/persistent-volumes.html">
+                    Stateful Applications Using Local Persistent Volumes
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/external-volumes.html">
+                    Stateful Applications Using Persistent External Volumes
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/recipes.html">
+                    Recipes
+                </a>
+            </li>
+        </ul>
+
+        <h5>Management</h5>
```

Comment:
To me, It's not really clear from the header whether this section is about managing Marathon or managing application deployments with marathon or something else. The current content is a mixture of _managing application deployments with Marathon_ (e.g. _Application Groups_, _Blue Green Deployments_, _Running Docker Containers_) and _Administrating Marathon_ (_SSL and Basic Auth_, _Framework Auth_, _Service Discovery_).

I'd put everything that covers managing applications and their deployments under _Using Marathon_ (see above), and the rest under _Configuring Marathon_ (or _Administrating Marathon_ if that's more specific).

Again: Does that make sense to you?